sql/parser/builtins.go

```diff
@@ -216,7 +222,7 @@ var builtins = map[string][]builtin{
 			return DString(strings.Trim(s, chars)), nil
 		}, DummyString),
 		stringBuiltin1(func(s string) (Datum, error) {
-			return DString(strings.Trim(s, " ")), nil
+			return DString(strings.TrimSpace(s)), nil
```

Comment:
This is a deviation from postgres, which only trims the space character, not other whitespace. I don't really care which definition we're using but if we're changing behavior it should be deliberate.